indi-sv305/sv305_ccd.cpp

```diff
@@ -277,6 +254,65 @@ bool Sv305CCD::Connect()
         pthread_mutex_unlock(&cameraID_mutex);
         return false;
     }
+    if (isDebug())
+    {
+        // Output camera properties to log 
+        IDLog("Camera Property:\n WxH= %ldx%ld, Color:%d, BayerPattern:%d, MaxBitDepth:%d, IsTriggerCam:%d\n",
+            cameraProperty.MaxWidth, cameraProperty.MaxHeight,
+            cameraProperty.IsColorCam,
+            cameraProperty.BayerPattern,
+            cameraProperty.MaxBitDepth,
+            cameraProperty.IsTriggerCam);
+        IDLog(" Bin:");
+        for (int i = 0; (i < (int)(sizeof(cameraProperty.SupportedBins)/sizeof(cameraProperty.SupportedBins[0]))) && cameraProperty.SupportedBins[i] != 0; i++) {
+            IDLog("%d ", cameraProperty.SupportedBins[i]);
+        }
+        IDLog("\n");
+        IDLog(" SupportedVideFormat:");
+        for (int i = 0; (i < (int)(sizeof(cameraProperty.SupportedVideoFormat)/sizeof(cameraProperty.SupportedVideoFormat[0]))) && cameraProperty.SupportedVideoFormat[i] != SVB_IMG_END; i++) {
+            IDLog("%d ", cameraProperty.SupportedVideoFormat[i]);
+        }
+        IDLog("\n");
+    }
+
+    // get camera properties ex
+    status = SVBGetCameraPropertyEx(cameraID, &cameraPropertyEx);
+    if (status != SVB_SUCCESS)
+    {
+        LOG_ERROR("Error, get camera property ex failed\n");
+        pthread_mutex_unlock(&cameraID_mutex);
+        return false;
+    }
+
+    if (isDebug())
```

Comment:
remove isDebug

Reply:
In line 257 you commented to rewrite to LOGF_DEBUG.
Is it correct that line 287 should be rewritten to LOGF_DEBUG as in line 257?

Reply:
Correct, LOGF_DEBUG can be used correctly without checking whether we are in debug or not.

Reply:
OK, I will replace from IDLog to LOGF_DEBUG